src/pyodide/internal/loadPackage.ts

```diff
@@ -43,57 +31,14 @@ function getPackageMetadata(requirement: string): PackageDeclaration {
   return obj;
 }
 
-// loadBundleFromR2 loads the package from the internet (through fetch) and uses the DiskCache as
-// a backing store. This is only used in local dev.
-async function loadBundleFromR2(requirement: string): Promise<Reader> {
-  // first check if the disk cache has what we want
-  const filename = getPackageMetadata(requirement).file_name;
-  let original = DiskCache.get(filename);
-  if (!original) {
-    // we didn't find it in the disk cache, continue with original fetch
-    const url = new URL(WORKERD_INDEX_URL + filename);
-    const response = await fetch(url);
-    if (response.status != 200) {
-      throw new Error(
-        `Could not fetch package at url ${url} received status ${response.status}`
-      );
-    }
-
-    original = await response.arrayBuffer();
-    DiskCache.put(filename, original);
-  }
-
-  if (filename.endsWith('.tar.gz')) {
-    const decompressed = await decompressArrayBuffer(original);
-    return new ArrayBufferReader(decompressed);
-  } else if (filename.endsWith('.tar')) {
-    return new ArrayBufferReader(original);
-  } else {
-    throw new Error('Unsupported package file type: ' + filename);
-  }
-}
-
-async function loadBundleFromR2WithRetry(
-  requirement: string,
-  currRetry: number
-): Promise<Reader> {
-  try {
-    return await loadBundleFromR2(requirement);
-  } catch (exc) {
-    if (currRetry < 3) {
-      console.warn(`Error loading '${requirement}' from R2: ${exc}`);
-      console.log('Retrying fetch in 5 seconds...');
-      await new Promise((r) => setTimeout(r, 5000));
-      return loadBundleFromR2WithRetry(requirement, currRetry + 1);
-    }
-
-    throw new Error(`Could not load '${requirement}' from R2: ${exc}`);
-  }
+// Helper function to get the path to a package in the Python package bucket
+function getPyodidePackagePath(version: string, filename: string): string {
+  return `python-package-bucket/${version}/${filename}`;
 }
```

Comment:
I don't think we need this function since it's a one-liner and only called in one place. I do like using the template string over `+`.